lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -258,6 +272,46 @@ def __init__(self, ufeSceneItem):
     def prependControlCreator(controlCreator):
         AETemplate._controlCreators.insert(0, controlCreator)
 
+    def orderSections(self, schemasAttributes):
+        '''
+        Choose the order in which the sections will be added to the AE template.
+        '''
+        availableSchemas = list(schemasAttributes.keys())
+
+        desiredFirstSchemas = [
+            'LightAPI',
+            '.* Light',
+            'lightLinkCollectionAPI',
+            'shadowLinkCollectionAPI',
+            'customCallbacks'
+        ]
+
+        desiredLastSchemas = [
+            'shader',
+            'transforms',
+            'display',
+            'extraAttributes',
+            'metadata',
+        ]
```

Comment:
Some of those list entries are not schema names but special names that are recognized when creating the UI, below.